doc-Infrastructure_Migration_Solution_Guide/cfme/doc/modules/con_Advanced_migration_option-_automating_pre-_and_post-migration_tasks_with_ansible.adoc

```diff
@@ -0,0 +1,6 @@
+[id="Advanced_migration_option-_automating_pre-_and_post-migration_tasks_with_ansible"]
+= Advanced Migration Option: Automating Pre- and Post-Migration Tasks with Ansible
+
+You can automate certain tasks before or after a virtual machine is migrated, for example, removing web server virtual machines from a load-balancing pool before migration and adding them back to the pool after migration. You create an Ansible service catalog item that contains the playbook and select it while creating the infrastructure migration plan.
```

Comment:
Hi Avital, I have some recommendations for this section:

1. Restructure the introduction sentence to mention Ansible playbooks. "{product-title} enables you to use Ansible playbooks to automate tasks on virtual machines before or after migration. For example, use a playbook to......"